documentation/slate/source/includes/disruptions.md

```diff
@@ -0,0 +1,267 @@
+Disruptions
+===========
+
+
+In this paragraph, we will explain how the disruptions are displayed in the different APIs.
+
+# Context
+
+## periods
+
+### production period of coverage data
+The production period is global to a navitia coverage and is the validity period of the coverage's data.
+This production period cannot exceed one year.
+
+### publication period of a disruption
+The publication period of a disruption is the period on which we want to display the disruption in navitia.
+
+The creator of the disruption might not want the traveller to know of a disruption before a certain date (because it's too uncertain, secret, ...)
+The publication period is the way to control this.
+
+### application periods of a disruption
+The application periods are the list of periods on which the disruption is active.
+For example it's the actual period when railworks are done and train circulation is cut.
+
+## Request date
+
+Request date is the navitia call date, the default value is now.
+
+# Summary
+
+To sum up we display an impact if 'now' is in the publication period.
+
+The status of the impact depends only of 'now' and is:
+
+* 'active' if 'now' is inside an application period
+* 'future' if there is an application period after 'now'
```

Comment:
Still that ambiguity : what's the status if we are inside an application period and there is another one later?